CruCentralCoast/Events/EventsCVC.swift

```diff
@@ -9,43 +9,100 @@
 import UIKit
 
 class EventsCVC: UICollectionViewController {
```

Comment:
Is this class being used anymore? I see that there is an EventsCVC and EventsTVC... Which is being used? Also, whichever is being used should be renamed to EventsVC